libp2pdht.nimble

```diff
@@ -10,7 +10,7 @@ skipDirs      = @["tests"]
 # Dependencies
 requires "nim >= 1.2.0",
          "nimcrypto >= 0.5.4 & < 0.6.0",
-         "bearssl#head",
+         "bearssl#f4c4233de453cb7eac0ce3f3ffad6496295f83ab",
```

Comment:
@dryajov example of what I mentioned that you can manually pin a commit version in a Nimble file.